packages/node_modules/@webex/plugin-meetings/test/unit/spec/members/index.js

```diff
@@ -212,5 +212,33 @@ describe('plugin-meetings', () => {
         assert.isRejected(members.raiseOrLowerHand('test1', true));
       });
     });
+
+    describe('#lowerAllHands', () => {
+      it('should fire spies correctly when lowerAllHands is called with valid params', async () => {
+        const members = createMembers({url: url1});
+
+        const {membersRequest} = members;
+
+        const generateOptionsSpy = sandbox.spy(MembersUtil, 'generateLowerAllHandsMemberOptions');
+        const lowerAllHandsMemberSpy = sandbox.spy(membersRequest, 'lowerAllHandsMember');
+
+        await members.lowerAllHands('test1');
+
+        assert.calledOnce(generateOptionsSpy);
+        assert.calledOnce(lowerAllHandsMemberSpy);
```

Comment:
issue: These should check what they are called with (to cover the util functions properly), possibly using `calledOnceWith`.